tests/testing.ts

```diff
@@ -251,12 +252,21 @@ export function runTests(
       } else {
         if (!preTestChecks(warpTestFolder)) return;
       }
-      filter = process.env.FILTER;
     });
 
     describe(`Running warp compilation tests on ${contractsFolder} solidity files`, async function () {
       findSolSourceFilePaths(warpCompilationTestPath, true).forEach((file) => {
-        if (filter === undefined || file.includes(filter)) {
+        let complexFiltering = filter === undefined && preFilters === undefined;
+        // if FILTER argument is passed, then only run tests that include the filter
+        // and preFilters are ignored, otherwise run tests that are in preFilters
+        if (filter !== undefined) {
+          complexFiltering = file.includes(filter);
+        }
+        if (filter === undefined && preFilters !== undefined) {
+          complexFiltering = preFilters.includes(file);
+        }
```

Comment:
I think you can change this if statement to:
```
        else if (preFilters !== undefined) {
          complexFiltering = preFilters.includes(file);
        }

Reply:
good point!